native-image-workshop/README.md

```diff
@@ -78,9 +86,9 @@ Did it work for you?
 
 ## Build a Native Image from the App
 
-Next, build a native image version of the application.
+Next, you are going to build a native image version of the application. This will start quicker than the Java application.
```

Comment:
... you are going to > we are going to

Packaging a Java application into a native image promises a much quicker startup than from a JAR file.

Reply:
updated